Backend/TimeTracker.Api/Controllers/TimerController.cs

```diff
@@ -0,0 +1,206 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using TimeTracker.Api.Database;
+using TimeTracker.Api.Database.Models;
+using TimeTracker.Api.DTOs;
+using TimeTracker.Api.Helpers;
+
+namespace Backend.Controllers
+{
+    [ApiController]
+    [Authorize]
+    [Route("[controller]")]
+    public class TimerController : ControllerBase {
+        MainDb database;
+        AuthHelper authHelper;
+
+        public TimerController(MainDb db, AuthHelper auth)
+        {
+            database = db;
+            authHelper = auth;
+        }
+
+        // returns a TimerDTO for the newly created Timer
+        [HttpPost]
+        [Route("Start")]
+        public async Task<GenericResponseDTO<TimerDTO>> StartTimer(TimerCreateDTO timerInfo) {
+
+            if(User == null) {
+                return new GenericResponseDTO<TimerDTO> {
+                    Message = "Not logged in.",
+                    Success = false
+                };
+            }
```

Comment:
This code is not really needed, it does not hurt anything though. The tests don't call this like an API endpoint so they ignore the `[Authorize]` decorator at the top of this file. In production use, this probably wouldn't happen.